src/js/components/Page.js

```diff
@@ -179,6 +180,23 @@ var Page = React.createClass({
     );
   },
 
+  getTopBanner() {
+    // todo use mountservice and mount LicenseBanner
```

Comment:
For testing I'd provide a short snippet in the PR's description

```javascript
MountService.registerComponent(
  <Banner>
    You have exceeded your 60 day limit by 2 days.
    {" "}
    <a href="https://mesosphere.com/product" className="message-link">
      More Information
    </a>
  </Banner>,
  "Page:TopBanner"
);
```

This way I could have merged the PR right away.